packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -2,6 +2,7 @@
   "running_tasks_title_text": "Running Tasks",
 
   "force_auth_web_login_authorize_dev_hub_text": "SFDX: Authorize a Dev Hub",
+  "force_auth_web_login_authorize_org_text": "SFDX: Authorize an Org",
```

Comment:
@vazexqi, it turns out I needed to install the prettier extension for vscode. It's wrapping the lines as expected now.

Reply:
@ruthemmanuelle, @ntotten  - Should this say "Authorize a Prod/Scratch Org" to make it more clear what kind of org it is that we are authorizing against?

Reply:
Does that mean that people can't authorize sandboxes? Is that because we don't prompt for the --instanceurl parameter? Do we intend to let them authorize sandboxes using this command in the future?

Reply:
They should be able to authorize a sandbox also. I feel like we should just keep it as org. Basically, there are orgs and dev hubs.

Reply:
Sorry I wrote wrongly. I meant "Authorize a Prod/Sandbox Org". The reason I was pedantic about this is because I don't want customers to use this command this to to authorize a scratch org. That's why I wanted to say Prod/Sandbox.

But, I'm OK with the current Authorize an Org.

If we don't see the instanceurl it will use the one from sfdx-project.json. But that's a good point about the instanceurl. We should see if we need to add that to the prompt. My inclination is not to. If they need that they can set the sfdx-project.json.